src/xenia/cpu/ppc/ppc_emit_memory.cc

```diff
@@ -732,8 +735,9 @@ int InstrEmit_stwcx(PPCHIRBuilder& f, const InstrData& i) {
 
   Value* ea = CalculateEA_0(f, i.X.RA, i.X.RB);
   Value* rt = f.ByteSwap(f.Truncate(f.LoadGPR(i.X.RT), INT32_TYPE));
-  f.Store(ea, rt);
-  f.StoreContext(offsetof(PPCContext, cr0.cr0_eq), f.LoadConstantInt8(1));
+  Value* res = f.ByteSwap(f.Truncate(f.LoadReserved(), INT32_TYPE));
+  Value* v = f.AtomicCompareExchange(ea, res, rt);
```

Comment:
coool - would be good to add a note to that effect, for others finding this code later - it looks like it should be doing the right thing, but because it's not (still under the lock) it could be confusing